api/services/administration/general/schemas/healthCheckUpdateRequestSchema.yaml

```diff
@@ -5,14 +5,41 @@ properties:
   healthCheck:
     type: object
     description: Group
-    required:
-      - unhealthyThreshold
-      - healthyThreshold
-      - interval
```

Comment:
Have you deleted the interval field? it is a required param with min of 10

Reply:
The field is not deleted. These 3 fields are not a required fields for update API, therefore removed.